vscode/tsconfig.json

```diff
@@ -0,0 +1,21 @@
+{
+	"compilerOptions": {
+		"module": "commonjs",
+		"target": "es6",
+		"outDir": "out",
+		"lib": [
+			"es6"
+		],
+		"sourceMap": true,
+		"rootDir": "src",
+		"strict": true   /* enable all strict type-checking options */
+		/* Additional Checks */
+		// "noImplicitReturns": true, /* Report error when not all code paths in function return a value. */
```

Comment:
Why are these checks commented out?

Reply:
@tobous  It was the default if irc

Reply:
If they are neither needed nor used I would suggest removing them.